lblod/ember-rdfa-editor-lblod-plugins
# GN-4436: review min max implementation of number plugin

### Overview
This PR introduces the usage of the `AuNativeInput` component and one-way-binding in the number variable widgets. 
The number-input, minimum and maximum fields have been updated to use `AuNativeInput` which uses the native `input` element instead of the ember `Input` component used by `AuInput`.

##### connected issues and PRs:
Jira: https://binnenland.atlassian.net/browse/GN-4436?atlOrigin=eyJpIjoiMzI2OWY1YzUzYTk5NDE4MDk1NmMxYzA2ZWI4OGI3NjciLCJwIjoiaiJ9

### How to test/reproduce
- Insert a number and check if input minimum/maximum values works.
- Check if updating a number-variable works.

### Challenges/uncertainties
A note on the usage of `type=number`: while chromium-based browsers support this input type well (they do not allow to input characters outside of numbers), firefox has a bug which allows inputting text in number inputs (see https://bugzilla.mozilla.org/show_bug.cgi?id=1398528). 
The question which stands: do we use inputs with type `text` everywhere (and show a warning to the user when the input contains characters outside of numbers), or do we use the type `number` and ignore the firefox bug?  



### Checks PR readiness
